apps/aecore/src/aec_peer_connection.erl

```diff
@@ -1091,22 +1091,24 @@ send_response(S, Type, Response) ->
     Msg = aec_peer_messages:serialize_response(Type, Response),
     send_msg(S, response, Msg).
 
-%% If the message is more than 65533 bytes it won't fit in a single Noise
-%% message and we need to fragment it.
+%% If the message is more than 65535 - 2 - 16 bytes (2 bytes for the length and
```

Comment:
65535 is 0xFFFF

Reply:
Yes, I was debating with myself if I was to change the representation.